docs/installation.rst

```diff
@@ -159,41 +132,55 @@ You'll need at least (use the cygwin installer to fetch/install these):
     liblz4_1 liblz4-devel  # from cygwinports.org
     git make openssh
 
-You can then install ``pip`` and ``virtualenv``:
-
-::
+You can then install ``pip`` and ``virtualenv``::
 
     easy_install-3.4 pip
     pip install virtualenv
 
-And now continue with the generic installation (see below).
+In case the creation of the virtual environment fails, try deleting this file::
 
-In case that creation of the virtual env fails, try deleting this file:
+    /usr/lib/python3.4/__pycache__/platform.cpython-34.pyc
 
-::
 
-    /usr/lib/python3.4/__pycache__/platform.cpython-34.pyc
+.. _pip-installation:
 
+Installation (pip)
+------------------
+
+Virtualenv_ can be used to build and install |project_name| without affecting
+the system Python or requiring root access.  Using a virtual environment is
+optional, but recommended except for the most simple use cases.
 
-Installation (pypi)
--------------------
+.. note::
+    If you install into a virtual environment, you need to **activate** it
+    first (``source borg-env/bin/activate``), before running ``borg``.
+    Alternatively, symlink ``borg-env/bin/borg`` into some directory that is in
+    your ``PATH`` so you can just run ``borg``.
 
-This uses the latest (source package) release from PyPi. ::
+This will use ``pip`` to install the latest release from PyPi::
 
     virtualenv --python=python3 borg-env
-    source borg-env/bin/activate   # always before using!
+    source borg-env/bin/activate
 
-    # install borg + dependencies into virtualenv
+    # install Borg + Python dependencies into virtualenv
     pip install 'llfuse<0.41'  # optional, for FUSE support
                                # 0.41 and 0.41.1 have unicode issues at install time
     pip install borgbackup
 
-.. note:: We install into a virtual environment here, but this is not a requirement.
+To upgrade |project_name| to a new version later, run the following after
+activating your virtual environment::
+
+    pip install -U borgbackup
 
 
 Installation (git)
 ------------------
 
+In addition to the dependencies installed above you will need Cython to compile
+the .pyx files to the respective .c files when using |project_name| code from
+git. If you want to run all tests, you also need fakeroot.  Both should be
+available through your package manager.
+
```

Comment:
Cython from the linux dist's packages might be rather old, maybe it is better to use pip install cython.